<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWaterLineExperiment</class>
 <widget class="QWidget" name="tabWaterLineExperiment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCommandView">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>generic Command View</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutPlotArea">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>plot window</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
